vllm/model_executor/kernels/linear/scaled_mm/helion.py

```
@@ -0,0 +1,396 @@
+                    in_dtype,
+                    missing,
+                )
+            return True, None
+
+        if missing:
+            return (
+                False,
+                f"has no pre-tuned config for weight shape (K={K}, N={N}) "
+                f"with in_dtype={in_dtype} at M={missing}. Set "
+                f"VLLM_HELION_LINEAR_SKIP_CONFIG_CHECK=1 to run anyway with "
+                f"the closest available config",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)